terraform/cloud-platform-components/templates/prometheus-operator.yaml.tpl

```diff
@@ -452,11 +515,9 @@ grafana:
     ## Secret must be manually created in the namespace
     ##
     tls:
-      - hosts:
-        - "${ grafana_ingress }"
-    # - secretName: prometheus-general-tls
-    #   hosts:
-    #   - prometheus.example.com
+    - secretName: prometheus-general-tls
```

Comment:
Does `secretName` need to be uncommented?
It was left as a comment so that the Grafana Ingress would use the default wildcard certificate. `prometheus-general-tls` is not being created anywhere?

Reply:
Hi @alkar - made changes - hopefully have got it right now?